src/plugins/intel_cpu/src/nodes/mkldnn_depth_to_space_node.cpp

```diff
@@ -157,7 +193,18 @@ void MKLDNNDepthToSpaceNode::createPrimitive() {
 
 void MKLDNNDepthToSpaceNode::prepareParams() {
     attrs.srcBlockedDims = getParentEdgeAt(0)->getMemoryPtr()->GetDescWithType<BlockedMemoryDesc>()->getBlockDims();
-    execPtr = std::make_shared<DepthToSpaceExecutor>(attrs);
+    auto builder = [](const DepthToSpaceKey& key) -> std::shared_ptr<DepthToSpaceExecutor> {
+        return std::make_shared<DepthToSpaceExecutor>(key.attrs);
+    };
+
+    DepthToSpaceKey key = {attrs};
```

Comment:
Now we can store the `key` instead of `attrs` in the node object, so that we can avoid this extra copy. Moreover, if we rewrite the `DepthToSpaceExecutor` from `DepthToSpaceAttrs` to `DepthToSpaceKey` we can skip this cumbersome access `key.attrs`. On the other hand, it is not necessary to name the key type with the `Key` word in the name. It this particular case we can just add `hash()` and `operator==` functions to the `DepthToSpaceAttrs` and reuse it as a key. It will simplify the code.

Reply:
@maxnick Applied review comments. Thx!